.gitignore

```diff
@@ -30,3 +30,6 @@ ci/packaging/suse/obs_files
 
 # ignore junit files
 ci/infra/testrunner/pr.xml
+
+# Backup files
+*~
```

Comment:
This should not be here. Instead have your own global gitignore for which you remove IDE files, please.
Example, set the content here in a `~./.gitignore` and put `core.excludesfile=~/.gitignore` in your global git config.

Reply:
Thanks, I didn't know about that!